### PR TITLE
ZCS-2703 : Fix L0 testcases of Calendar app using Firefox browser

### PR DESCRIPTION
**Pull request by swapnilpingle89**
Corrected verification data as per firefox browser and added condition to select data based on browser being used for execution.

### CODE REVIEW

**Review by shubhamvitec — APPROVED** [review]

**Comment by jiteshsojitra** [comment]
Thanks Swapnil.

1. Can you change review request subject like ZCS-bug: summary.

2. Can you replace if(ConfigProperties.getStringProperty("browser") with if (ConfigProperties.getStringProperty("browser")?

3. Does these changes works with Chrome as well?

After doing above changes, please do changes un Universal UI project as well.

File: src/java/com/zimbra/qa/selenium/projects/ajax/ui/calendar/FormApptNew.java [hunk 1]
```diff
@@ -999,7 +999,11 @@ public void zFillField(Field field, String value) throws HarnessException {
 
 			if (field == Field.StartDate || field == Field.EndDate || field == Field.StartTime
 					|| field == Field.EndTime) {
-				this.sMouseDownAt(locator, "");
+
```
Comment (jiteshsojitra) [inline src/java/com/zimbra/qa/selenium/projects/ajax/ui/calendar/FormApptNew.java]: Remove this empty line too.

File: src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/viewinvite/ViewInviteWithDisplayMailPreference.java [hunk 2]
```diff
@@ -51,14 +51,21 @@ public ViewInviteWithDisplayMailPreference() throws HarnessException {
 	)
 	public void ViewHTMLInviteWithHTMLPreference_01() throws HarnessException {
 		
+		String fontfamilyData="";
```
Comment (jiteshsojitra) [inline src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/viewinvite/ViewInviteWithDisplayMailPreference.java]: Minor, please replace String fontfamilyData=""; to String fontFamilyData = ""; Use camelcase variable name,

**Review by jiteshsojitra — APPROVED** [review]

**Comment by jiteshsojitra** [comment]
Will it fix all L0 failed testcases of Calendar for Firefox according to review request summary?

File: src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/viewinvite/ViewInviteWithDisplayMailPreference.java [hunk 3]
```diff
@@ -51,14 +51,21 @@ public ViewInviteWithDisplayMailPreference() throws HarnessException {
...
+		
 		// Import appointment using text mime
 		final String subject = "multiline HTML body";
 		final String boldContent = "<div><strong>BoldString</strong></div>";
 		final String italicContent = "<div><em>ItalicString</em></div>";
 		final String underlineContent ="<div><span style=\"text-decoration: underline;\">Underline String</span></div>";
 		final String colorFontContent ="<div><span style=\"color: rgb(255, 0, 0);\">Red Color text</span></div>";
 		final String colorBackgroundContent ="<div><span style=\"background-color: rgb(0, 128, 0);\">Green Background</span></div>";
-		final String fontFamilySizeContent ="<br /><div><span style=\"font-family: &quot;comic sans ms&quot;, &quot;comic sans&quot;, sans-serif; font-size: 14pt;\">Number list below :</span>�</div><br />";
+		final String fontFamilySizeContent = fontfamilyData;
```
Comment (jiteshsojitra) [inline src/java/com/zimbra/qa/selenium/projects/ajax/tests/calendar/meetings/organizer/singleday/viewinvite/ViewInviteWithDisplayMailPreference.java]: Remove one of the variable, both are doing same. fontFamilySizeContent = fontfamilyData;. If you use fontfamilyData name then please use camel case variable name for e.g. fontFamilyData.

**Review by jiteshsojitra — COMMENTED** [review]
Thanks Swapnil for making the changes for universal as well.

File: src/java/com/zimbra/qa/selenium/projects/ajax/ui/calendar/FormApptNew.java (hunk 1 above)
Comment (swapnilpingle89) [reply src/java/com/zimbra/qa/selenium/projects/ajax/ui/calendar/FormApptNew.java]: Removed empty line

**Review by miteshsavani — APPROVED** [review]

**Pull request merged**